test/_/release-dst/frontend/index.js

```diff
@@ -1 +1,64 @@
-(function(){'use strict';0,0,0;var a=function(b,c=200){var d;return function(...f){d||(d=setTimeout(()=>{b.apply(this,f),d=!1},c))}}(()=>{console.log('other mod')});0,0})();
\ No newline at end of file
+(function(){'use strict';var k=function(p){'use strict';function q(B,C){if(1===arguments.length){for(var D=B[0],E=1;E<B.length;E++)D=q(D,B[E]);return D}for(var F in C)if('class'===F){var G=B[F]||[];B[F]=(Array.isArray(G)?G:[G]).concat(C[F]||[])}else if('style'===F){var G=u(B[F]),H=u(C[F]);B[F]=G+(G&&H&&';')+H}else B[F]=C[F];return B}/**
+   * Process array, object, or string as a string of classes delimited by a space.
```

Comment:
If not, [shouldPrintComment](https://github.com/babel/babili/issues/396) must do.